OrderTest/Scenes/Models/Order.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ProductResponse.swift
+//  Drawing
+//
+//  Created by Iulian Florea on 7/16/21.
+//
+
+import Foundation
+
+struct Order: Codable, Identifiable {
+    let id: Int
+    let productDescription: String
+    let price: Int
+    let deliverTo: DeliverTo
+    let imageURL: String
+    let status: String
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
Nice work with deserialization 👍 I would have probably chosen a different name for "deliverTo", but this is a minor thing, the CodingKeys part is the one that counts.